### DISCUSSION TITLE
Token decrypt?

### DISCUSSION THREAD

**Discussion by Lizzard44** [teslamate-org/teslamate#2877] [Q&A]
I would like to use the TeslaMate token for my application.
Unfortunately I'm not that fit in Elixir to decrypt the encrypted token from the database.

I think it's done somewhere in TeslaMate. But don't find it.

Is there an OutOfTheBox program that could do this?

**Comment by LelandSindt** [comment]
This is how we did it in https://github.com/tobiasehlert/teslamateapi

https://github.com/tobiasehlert/teslamateapi/commit/3f9e8c471bf2477ad7827cc208b9f492f4b0bc38
